homeassistant/components/dexcom/config_flow.py

```diff
@@ -25,7 +32,9 @@ class DexcomConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
         """Handle the initial step."""
         errors = {}
```

Comment:
Missed

Reply:
I only set the ones that were compulsory for mypy (in init)
These are not so I ignored them...
I'm not sure about these new ones you mention...

Reply:
Ah check